packages/harpguru-core/src/utils/reduce-for-new-column-bounds/determine-next-column-bounds.ts

```diff
@@ -0,0 +1,21 @@
+import { ZoomIds } from '../../types'
+
+import { determineBest7Holes } from './determine-best-7-holes'
+
+export const determineNextColumnBounds = (
+  harpLength: number,
+  currentColumnBounds: 'FIT' | readonly [number, number],
+  zoomId: ZoomIds
+): 'FIT' | readonly [number, number] => {
+  return (() => {
+    if (zoomId === ZoomIds.Fit) return 'FIT'
+    if (currentColumnBounds === 'FIT') {
+      if (zoomId === ZoomIds.Seven) return [0, 6] as const
```

Comment:
It would be good to be able to make this a simple `return [0, zoomId]`

If the value of the ZoomIds enum is `'FIT' | number` then I think that should be possible.

Reply:
Done